dali/python/nvidia/dali/fn.py

```diff
@@ -89,6 +89,7 @@ def fn_wrapper(*inputs, **kwargs):
     fn_wrapper.__name__ = wrapper_name
     fn_wrapper.__qualname__ = wrapper_name
     fn_wrapper.__doc__ = wrapper_doc
+    fn_wrapper._schema_name = op_class.schema_name
```

Comment:
I'd suggest passing the op_class, not just the name - that way we'd have access to whatever properties of the underlying op class we want.
```suggestion
    fn_wrapper._op_class = op_class
```
Also, please add it also to op_wrapper, too. I think this code needs some cleanup after the introduction of debug mode... :\

Reply:
Probably this change or anything related should be made it separate PR not during docs rework.

I am a bit worried about tying the whole class to the fn functions, but those are private members, and who checks the members of the functions?